.buildconfig.yml

```
@@ -371,6 +371,10 @@ projects:
     path: components/support/webextensions
     description: 'A component containing building blocks for features implemented as web extensions.'
     publish: true
+  lib-auth:
+    path: components/lib/biometric-prompt
+    description: 'Component for authentication using biometric'
```

nit: wording. Since we're increasing the scope of this component, we should describe it as such.
```suggestion
    description: 'A component for various kinds of authenticating mechanisms.'
```

---

Done